lib/delta/op.ex

```diff
@@ -157,7 +160,11 @@ defmodule Delta.Op do
     {op1, a, op2, b}
   end
 
-  defp take_partial(%{"insert" => text} = op, len) do
+  def take_partial(op, len, opts \\ [])
```

Comment:
take_partial should remain private. Also when doing so it reveals the default `opts \\ []` is not used so this line can be removed.